ethstorage/p2p/protocol/peer.go

```diff
@@ -21,13 +21,15 @@ type Peer struct {
 	direction   network.Direction
 	version     uint                        // Protocol version negotiated
 	shards      map[common.Address][]uint64 // shards of this node support
+	tracker     *Tracker
```

Comment:
Geth snap sync uses Trackers to calculate the mean value of all existing peers as the initial value for a newly joined peer. Do we want to follow the same practice?

Reply:
I don't think this is very necessary, because it is just an initial value, and it will be updated to the appropriate value soon. Run the es-node on AX101 using the 4M as p2p.request.size and the peer's capacity will be greater than our maximum value of 8M. 

![28849da6e66d19040145dfc061b1d84](https://github.com/ethstorage/es-node/assets/14110556/2628f218-28ec-42ef-97b1-d9c82ae509f8)
t.capacity = (1-measurementImpact)*(t.capacity) + measurementImpact*measured
these 2 peers have it capacity larger than 16M and 32M, 
If p2p.request.size = 4M  (capacity = 4M/ 8s = 0.5M)
    16 * 0.1 + 0.5 * 0.9 > 1M
If p2p.request.size = 1M  (capacity = 4M/ 8s = 0.125M)
    16 * 0.1 + 0.125 * 0.9 > 1M
For peers with good network conditions, the initial value does not have much impact.

For peers with bad network conditions, each timeout request reduces capacity to 90% of the previous one.

So, change the run.sh by removing p2p.request.size setting, and the initial request size changed to 1M.